backend/src/main/java/com/webappsbusters/parcelmanagement/service/DetermineSize.java

```diff
@@ -0,0 +1,23 @@
+package com.webappsbusters.parcelmanagement.service;
+import com.webappsbusters.parcelmanagement.domain.*;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class DetermineSize {
+    public ParcelSize DetermineDummyParcelSize(int length, int width, int height) {
```

Comment:
please rename to calculateParcelSize

Reply:
not sure, whole service name or just the method?

Reply:
changed the method only